src/views/NewFlow.vue

```diff
@@ -57,22 +60,42 @@ class NewFlow extends Vue {
   @Prop({default: () => ({})}) readonly appConfig!: object
   @Prop({default: () => ({})}) readonly builderConfig!: object
 
+  didUserSubmit = false
+
+  @Watch('activeFlow', {deep: true, immediate: true})
+  async onActiveFlowChanged(newFlow: IFlow) {
+    if (newFlow) {
+      console.debug('watch/activeFlow:', 'active flow has changed from new flow flow page, validating ...');
+      await this.validate_flow({flow: newFlow})
+    }
+  }
+
+  get createFlowTitle(): string {
+    return this.$store.state.trees.ui.title.createFlow
+  }
+
   async mounted(): Promise<void> {
     await this.flow_addBlankFlow()
   }
 
-  async created(): Promise<void> {
+  async beforeCreate(): Promise<void> {
     const {$store} = this
 
-    forEach(store.modules, (v, k) =>
-      !$store.hasModule(k) && $store.registerModule(k, v))
+    forEach(store.modules, (v, k) => !$store.hasModule(k) && $store.registerModule(k, v))
```

Comment:
had to extract module registration in `beforeCreate` to fix refresh on `flows/new` route

Reply:
otherwise we had something like here https://github.com/vuejs/vuex/issues/1622#issuecomment-536960343